packages/constants/src/mcf.ts

```diff
@@ -438,3 +440,700 @@ export const SSEC_EQA_LIST: ISsecEqa[] = [
+  GENERAL_MANAGEMENT: `<h1>Apply for General Manager Jobs in Singapore</h1><p>General management refers to the diverse range of responsibilities required to keep an organisation running. With an increasing number of businesses in the market, general management will become a crucial aspect of every company and the demand for such jobs will indeed see a rise. As a general manager, you will be able to set forth on an exciting profession and the opportunities for jobs in this field are endless in Singapore.</p><h2>Role of a General Manager</h2> <p>General managers are tasked to 
```

Comment:
Is there a scenario where `processedJobTitle` or `processedCompany` is null? Because for example if both are null this will return `nullnull` right?

Reply:
By default, if there is no jobTitle or company, it will just return the jobId for the url. But I'm not sure to understand what exactly you're talking about ?

Reply:
Oh I was wondering if `processedJobTitle` or `processedCompany` could be null, which will then cause the `urlSegment` to become "nullnull"

Reply:
if there is no such possible scenario then it's fine